x/concentrated-liquidity/lp.go

```diff
@@ -428,12 +428,11 @@ func (k Keeper) isLockMature(ctx sdk.Context, underlyingLockId uint64) (bool, er
 		return false, err
 	}
 
-	// Check if the lock has expired
-	if underlyingLock.EndTime.After(ctx.BlockTime()) {
-		// Lock is still active, so we cannot withdraw from this position
+	if underlyingLock.EndTime.IsZero() {
+		// Lock is still active, so we can't withdraw from this position
 		return false, nil
```

Comment:
This branch is never hit with tests

Reply:
I believe it is here

https://github.com/osmosis-labs/osmosis/blob/f8e30645c9f38a7f3315fd2eb8fc6c505aa40d92/x/concentrated-liquidity/lp_test.go#L1150-L1154